src/workerd/api/modules.h

```diff
@@ -15,6 +17,8 @@ namespace workerd::api {
 template <class Registry>
 void registerModules(Registry& registry, auto featureFlags) {
   node::registerNodeJsCompatModules(registry, featureFlags);
+  pyodide::registerPyodideModules(registry, featureFlags);
+  registerUnsafeModules(registry, featureFlags);
```

Comment:
This makes `UnsafeEval` available only to builtin/internal modules. So I don't think this change by itself is risky: by itself it makes `UnsafeEval` accessible in various builtin things but none of them use it so it's dead code. OTOH we are using it somewhere so we have to be careful about that.

I can gate it on experimental too.

Reply:
I think a mere existence of `UnsafeEval` was controversial. I'd like @jasnell to chime in: is it ok to always register UnsafeEval?

> I can gate it on experimental too.

I think it would be best if you installed it only if python modules are installed (which is already gated?)

Reply:
> Does this make the unsafe eval module usable by all workers?

No, as long as it is registered as `INTERNAL` it is only accessible by our own built-ins.

> I think a mere existence of `UnsafeEval` was controversial

Not so much. It just shouldn't be accessible to regular worker code, which it is not.

Reply:
Okay I added some tests for this.